packages/main-ui/src/pages/deliberations/controller.rs

```diff
@@ -125,6 +125,24 @@ impl Controller {
         self.mouse_pos.with(|v| v.1 + 20.0)
     }
 
+    pub fn move_user_page(&self, project_id: i64) {
+        use web_sys::window;
+
+        let lang = self.lang;
+        let url = crate::config::get().user_url;
+
+        let user_domain = format!(
+            "{}/{}/projects/{}",
+            url,
+            if lang == Language::Ko { "ko" } else { "en" },
```

Comment:
```suggestion
            self.lang,
```
Language might implement Display trait

Reply:
넵 수정했습니다!